src/ui/public/kbn_top_nav/bread_crumbs/bread_crumb_urls.js

```diff
@@ -1,3 +1,4 @@
+import _ from 'lodash';
 /**
  * @typedef BreadCrumbUrl {Object}
  * @property breadcrumb {String} a breadcrumb
```

Comment:
Can you add path property here and highlight the difference between path and url?  Url is full url, path is the portion of the url that matches the breadcrumb (or something like that).